blog/content/post/2020-04-30-ipxe-https.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Enabling HTTPS support in iPXE
+abstract: 'Abstract first sentence.
+          Abstract second sentence.
+          Abstract third sentence.'
+cover: /covers/image-file.png
+author: michal.zygowski
+layout: post
+published: true
+date: 2020-04-30
+archives: "2020"
+
+tags:
+  - firmware
+  - security
+  - coreboot
+categories:
+  - Firmware
+  - Security
+  - coreboot
+  - iPXE
```

Comment:
@miczyg1 i would prefer to create specific tags rather than new categories, like `iPXE`

Reply:
Sorry about that, I thought tags are fixed, but not categories. Moved iPXE to tags